src/BinSkim.Driver/BinSkim.Driver.csproj

```diff
@@ -28,7 +28,7 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.7.82" />
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.1.0" />
```

Comment:
>Microsoft.CodeAnalysis [](http://example.com/codeflow?start=31&length=22)

@michaelcfanning , I was taking a look and I saw that the code that uses this reference was created as a PoC for Roslyn.

I was thinking in removing and cleaning-up the code.